cli/sync.ts

```diff
@@ -7,84 +7,95 @@ import { normalizePackageName } from "../util/mod.ts"
 import { tempDir } from "../util/tempDir.ts"
 import { resolveNets } from "./resolveNets.ts"
 
+const descriptionLeading = "Sync net specs and update your "
+
+const syncNode = withCommonOptions(new Command()
+  .description(`${descriptionLeading} \`package.json\``))
+  .action(runSyncNode)
+
+const syncDeno = withCommonOptions(new Command()
+  .description(`${descriptionLeading} import map`))
+  .option("--import-map <import-map:string>", "the import map path", {
+    default: "import_map.json",
+  })
+  .action(runSyncDeno)
+
 export const sync = new Command()
-  .type("runtime", new EnumType(["deno", "node"]))
   .description("Sync net specs and update your manifest")
-  .arguments("<runtime:runtime>")
-  .option("-n, --nets <nets:file>", "nets.ts file path", { default: "./nets.ts" })
-  .option("--check", "ensures that metadata and codegen are in sync")
-  .option("-o, --out <out:string>", "Metadata and codegen output directory", {
-    default: "target/capi",
-  })
-  .option("-s, --server <server:string>", "")
-  .option(
-    "--runtime-config <runtimeConfig:string>",
-    "the import_map.json or package.json file path",
-  )
-  .action(runSync)
+  .command("node", syncNode)
+  .command("deno", syncDeno)
```

Comment:
whit the subcommand approach, running `capi sync` does nothing.
Perhaps it should print a help message

Reply:
or we could have 2 commands `sync-deno` and `sync-node`; or go back to the runtime argument
```
.arguments("<runtime:runtime>")

Reply:
> with the subcommand approach, running capi sync does nothing.

Is there an easy way to get it to print out the help text? Surprised cliffy doesn't do this by default.

Reply:
adding an `.action` before the subcommand seems to do it
```ts
  .description("Sync net specs and update your manifest")
  .action(function() {
    console.log("help")
    this.showHelp()
  })
  .command("node", syncNode)
  .command("deno", syncDeno)
```